string measurment
string sensor
string value